<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TQMQ11K" failures="1" tests="3" name="LnkedTest" time="20.736" errors="0" timestamp="2022-12-30T14:05:42 IST">
  <testcase classname="testngbasicprograms.LinkedTestDemo" name="doLogin" time="3.182"/>
  <testcase classname="testngbasicprograms.LinkedTestDemo" name="inputData" time="13.456"/>
  <testcase classname="testngbasicprograms.LinkedTestDemo" name="signOut" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [(44) Feed | LinkedIn] but found [(43) Feed | LinkedIn]">
      <![CDATA[java.lang.AssertionError: expected [(44) Feed | LinkedIn] but found [(43) Feed | LinkedIn]
at testngbasicprograms.LinkedTestDemo.signOut(LinkedTestDemo.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- signOut -->
</testsuite> <!-- LnkedTest -->
